### PR TITLE
changes BottomAppBar background color

### PR DESCRIPTION
**Pull request by Raynafs**
# Scope

_Please make sure to read the [Contribution Guidelines](https://github.com/droidconKE/droidconKE2023Android/blob/main/CONTRIBUTING.md)
and check that you understand and have followed it as best as possible Explain what your feature
does in a short paragraph._ please check the below boxes
- [x] I have followed the coding conventions
- [x] I have added/updated necessary tests
- [x] I have tested the changes added on a physical device
- [x] I have run `./codeAnalysis.sh` to make sure all lint/formatting checks have been done.

## Closes/Fixes Issues
_Declare any issues by typing `fixes #1` or `closes #1` for example so that the automation can kick
in when this is merged_

## Other testing QA Notes
_What have you tested specifically and what possible impacts/areas there are that may need retesting
by others._

Please add a screenshot (if necessary)

### CODE REVIEW

**Comment by misshannah** [comment]
Hi @Raynafs,

Please run ./codeAnalysis.sh then let us know. Also, adding screenshots in your PR would be so cool!

File: presentation/src/main/java/com/android254/presentation/common/bottomnav/BottomNavigationBar.kt [hunk 1]
```diff
@@ -56,13 +55,23 @@ fun BottomNavigationBar(navController: NavHostController) {
...
+                        selectedTextColor = MaterialTheme.colorScheme.secondary,
+                        unselectedTextColor = MaterialTheme.colorScheme.onBackground,
+                        indicatorColor =  MaterialTheme.colorScheme.background.copy(alpha = 0f)
+                    )
+                    else -> NavigationBarItemDefaults.colors(
+                        selectedIconColor = MaterialTheme.colorScheme.primary,
+                        unselectedIconColor = MaterialTheme.colorScheme.onBackground,
+                        selectedTextColor = MaterialTheme.colorScheme.secondary,
+                        unselectedTextColor = MaterialTheme.colorScheme.onBackground,
+                        indicatorColor =  MaterialTheme.colorScheme.background.copy(alpha = 0f)
```
[the remark this answers is not in the document]
Comment (Raynafs) [reply presentation/src/main/java/com/android254/presentation/common/bottomnav/BottomNavigationBar.kt]: Hi yes sure. I'm working on it.

**Comment by Raynafs** [comment]
WhatsApp Image 2023-04-17 at 18 07 03
WhatsApp Image 2023-04-17 at 18 06 25



So I've updated the code but now the only problem is that the primary color for the dark theme isn't green so the icon color can't be green.

This is how the dark theme is expected to look

**Comment by Raynafs** [comment]
So I guess the chai themes should be updated?

File: README.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ App will have the following features:
 ## Designs
 
 This is the link to the app designs:  
-[Light Theme] (https://xd.adobe.com/view/dd5d0245-b92b-4678-9d4a-48b3a6f48191-880e/)  
+[Light Theme] (https://xd.adobe.com/view/dd5d/codeAnalysis.sh0245-b92b-4678-9d4a-48b3a6f48191-880e/)
```
Comment (wangerekaharun) [inline README.md:69]: You shouldn't change this

File: chai/src/main/java/com/droidconke/chai/atoms/Color.kt [hunk 3]
```diff
@@ -202,7 +202,7 @@ val md_theme_dark_onSurfaceVariant = Color(0xFF45454f)
  * DARK SURFACE TINT
  */
 val md_theme_dark_surfaceTint = Color(0xFF7DE1C3)
-val md_theme_dark_surfaceTintColor = Color(0xFF7DE1C3)
+
```
Comment (wangerekaharun) [inline chai/src/main/java/com/droidconke/chai/atoms/Color.kt:205]: Don't remove this

File: README.md (hunk 2 above)
Comment (Raynafs) [reply README.md]: Oh sorry. I've corrected both

File: presentation/src/main/java/com/android254/presentation/common/bottomnav/BottomNavigationBar.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (wangerekaharun) [reply presentation/src/main/java/com/android254/presentation/common/bottomnav/BottomNavigationBar.kt]: @Raynafs did you look at this as well?

[the remark this answers is not in the document]
Comment (Raynafs) [reply presentation/src/main/java/com/android254/presentation/common/bottomnav/BottomNavigationBar.kt]: Yes I did. I changed it on 17th April, And then sent screenshots as well. The only challenge is with the dark mode primary color.

**Review by wangerekaharun — APPROVED** [review]

File: presentation/src/main/java/com/android254/presentation/common/bottomnav/BottomNavigationBar.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (wangerekaharun) [reply presentation/src/main/java/com/android254/presentation/common/bottomnav/BottomNavigationBar.kt]: Got it

**Review by chepsi — APPROVED** [review]

**Pull request merged**